addons/website/static/src/scss/website.scss

```diff
@@ -543,6 +543,9 @@ table.table_desc tr td {
     }
     // Content
     .carousel-inner {
+        // Create a new stacking context to ensure that elements like indicators
+        // and arrows will always appear on top of the carousel content.
+        z-index: 0;
```

Comment:
I don't like using the z-index property in the sole purpose of creating a new stacking context:
- It only works if the element has `position: relative` too (which is only the case here thanks to the bootstrap rule but so, it is quite fragile)
- You don't actually want to change the z-index of the element: it can mess up with the element's own stacking order.

In general, I preferred any of the alternative that works. The most ancient and known one being `opacity: 0.999` which does not actually change the opacity of the element (because the browser has no such precision), creates a stacking context and is rarely used on whole containers.

But now, I see there is an even better alternative: https://developer.mozilla.org/en-US/docs/Web/CSS/isolation :partying_face: Let's use that :)

Reply:
> But now, I see there is an even better alternative: https://developer.mozilla.org/en-US/docs/Web/CSS/isolation partying_face Let's use that :)

@qsm-odoo I made the change, thank you I didn't know this rule..